addons/barcodes_gs1_nomenclature/data/barcodes_gs1_rules.xml

```diff
@@ -174,7 +174,7 @@
             <field name="sequence">314</field>
             <field name="encoding">gs1-128</field>
             <field name="pattern">(314[0-5])(\d{6})</field>
-            <field name="associated_uom_id" ref="uom.uom_square_meter"/>
+            <field name="associated_uom_id" ref="uom.product_uom_square_meter"/>
```

Comment:
Not sure if it's worth renaming the IDs in this commit/PR to make it consistent with the other ones. It requires us to check _really_ carefully that the change is correctly applied everywhere since it will only break at runtime when it can't find the ID